cli/tests/integration_tests.rs

```diff
@@ -2015,6 +2015,12 @@ itest!(ts_import_from_js {
   http_server: true,
 });
 
+itest!(jsx_import_from_ts {
+  args: "run --quiet --reload jsx_import_from_ts.ts",
+  output: "jsx_import_from_ts.ts.out",
+  http_server: true,
+});
+
```

Comment:
Run this test case on master and it fails with: `error: No such file or directory (os error 2)`